packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -254,16 +254,25 @@ class RegisteredServiceExtensionButton {
   }
 }
 
-bool tabDisabledByQuery(String key) {
+Set _hiddenPages;
```

Comment:
Make these all
Set<String>

Reply:
I'm guessing you meant add the type? Changed to `Set<String>`